mentalmodel.md

```diff
@@ -0,0 +1,94 @@
+
+We decided to ignore  leap seconds since they are arbitrarily decided upon and unpredictable. An `Instant` can therefore not be said to be on the TAE timeline, but rather on the UTC timeline. In terms of leap-seconds we accept them a input (`fromString`) but immediately convert them to the relevant numeric representation and therefore ignoring the fact that this came from a leap-second.
+
+Because `Instant` is just a point in time, it has no concept of calendars or time-units as we use in civil society, the `Instant` objects do not have members representing these either.
+
+Conceptually `Instant`s are not geopgraphically tied to any place. They are equally valid on Mars and Venus as on earth.
+
+## OffsetDateTime
+
+`OffsetDateTime` objects represent a point in time together with a specific offset from UTC. Given that these offets are only valid on Earth, they are tied to a place. However they do not have a timezone or dayligh-davings adjustments or anything of that nature.
+
+As such they can be considered at best to be tied to the planet Earth rather than a specific locality. However because they are tied to the planet, they do have a concept of time units uch as hours and days. And because we restrict ourselves to the proleptic gregorian calendar, they do have months and years that correspond with our western understanding of these ideas.
+
+## ZonedDateTime
+
+`ZonedDateTime` objects represent a point in time in a specific IANA timezone. Since these timezones are tied to a specific locality (city/country), these are more precise and observe daylight savings changes.
+
+They also have a concept of time units and are most like what the current `Date` object provides, with the exception of the timezone being a specified value rather that magically intuited from system information.
+
+## CivilDateTime
+
+Contrary to `ZonedDateTime` and `OffsetDateTime` the `CivilDateTime` objects are not tied to an absolute point in time. An example for this would be the statement *the next new year is at 2020-01-01T00:00:00.000000000*. This is true independent of where on the planet one is and independent of any locality. It also represents a different point in time when combined with a timezone:
+
+ * Berlin: `2019-12-31T23:00:00.000000000Z`
+ * London: `2020-01-01T00:00:00.000000000Z`
+ * New York: `2020-01-01T05:00:00.000000000Z`
+
+## CivilDate
+
+This contain just the date part. It can be mentally considered the *leaf of a desk-calendar*. It can be placed anywhere on the planet, can mean any time within the 24-hour day, but the extent of the time period cannot be acurately specified as it has a different meaning in Australia and Europe for example.
```

Comment:
```suggestion
This contains just the date part. It can be mentally considered the *leaf of a desk-calendar*. It can be placed anywhere on the planet, can mean any time within the 24-hour day, but the extent of the time period cannot be acurately specified as it has a different meaning in Australia and Europe for example.
```